website/docs/terms/json.md

```diff
@@ -0,0 +1,103 @@
+
+- `menu_items[0]` will return “cheesecake”
+- `menu_items[1]` will return “danish”
+- `menu_items[2]` will return “coffee”
+
+Dictionaries and arrays can be nested in JSON objects as well as nested in each other. **Dictionaries and arrays can only be values. They can never be keys.**
+
+Here’s an example of a JSON object describing a tweet from [Twitter’s developer platform](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/overview).
+
+```json
+tweet = 
+{
+  "created_at": "Thu Apr 06 15:24:15 +0000 2017",
+  "id_str": "850006245121695744",
+  "text": "1\/ Today we\u2019re sharing our vision for the future of the Twitter API platform!\nhttps:\/\/t.co\/XweGngmxlP",
+  "user": {
+    "id": 2244994945,
+    "name": "Twitter Dev",
+    "screen_name": "TwitterDev",
+    "location": "Internet",
+    "url": "https:\/\/dev.twitter.com\/",
+    "description": "Your official source for Twitter Platform news, updates & events. Need technical help? Visit https:\/\/twittercommunity.com\/ \u2328\ufe0f #TapIntoTwitter"
+  },
+  "place": {   
+  },
+  "entities": {
+    "hashtags": [      
+    ],
+    "urls": [
+      {
+        "url": "https:\/\/t.co\/XweGngmxlP",
+        "unwound": {
+          "url": "https:\/\/cards.twitter.com\/cards\/18ce53wgo4h\/3xo1c",
+          "title": "Building the Future of the Twitter API Platform"
+        }
+      }
+    ],
+    "user_mentions": [     
+    ]
+  }
+}
+```
+
+Here's a quick quiz to see if you're understanding the file's structure:
+
+<details>
+<summary>How would you call the user ID?</summary>
+tweet[‘user’][‘id’]
+</details>
+
+<details>
+<summary>How would you call the unwound url?</summary>
+Tweet[‘entities’][‘urls’][0][‘unwound’][‘url’]
+</details>
+
+As you can see, JSON objects can get complex pretty quickly.
+
+## Why is JSON important in modern analytics?
+
+Semi-structured data offers flexibility with the trade-off of being more complex. JSON doesn’t require a pre-defined schema. It allows nesting, values can be different data types, and it lends itself well to changes in the shape of the incoming data. As you can imagine, the above Tweet object would look very different if we tried to restructure it so it could fit into a table. It would be hard to read or we would lose information or both. 
+
+## Use cases for JSON
+
+JSON is lightweight and often used to transfer data over a network connection. As we’ve seen, data from social media sites are often stored as JSON objects. JSON is also commonly how data from IoT sensors is formatted. You’ll often see JSON when using an API.
+
+## Conclusion
+
+The greatest strength of JSON also acts as its weakness— the data it contains informs the shape the object takes, rather than the other way around. Structured data is the bread and butter of data, but a semi-structured format is an alternative option when a tabular format becomes too rigid to describe the relationships between different entities. 
```

Comment:
```suggestion
The greatest strength of JSON also acts as its weakness—the data it contains informs the shape the object takes, rather than the other way around. Structured data is the bread and butter of analytics work, but a semi-structured format is an alternative option when a tabular format becomes too rigid to describe the relationships between different entities. 
```